### ISSUE TITLE
Support TypeScript 4.x

### ISSUE THREAD

**Issue by krisztianb** [KyleMayes/eslint-plugin-ordered-imports#8]
Hi Kyle,

thanks for your great plugin. I've just migrated from TSLint to ESLint. The simple ordered-imports rule of ESLint is not sufficient for replacing the TSLint one. But I found that your plugin does exactly what is needed. Great job!

My project is using the latest TypeScript version (currently 4.0.3). Your package defines the following peer dependencies in your [package.json](https://github.com/KyleMayes/eslint-plugin-ordered-imports/blob/master/package.json) file:

```javascript
"peerDependencies": {
    "eslint": "^7.0.0",
    "typescript": "^3.7.0"
  }
```

Could you please add support for TS 4.x? That would remove the following warning I get when installing your package:

> npm WARN eslint-plugin-ordered-imports@0.3.0 requires a peer of typescript@^3.7.0 but none is installed. You must install peer dependencies yourself.

You can find an example on how this could be achieved [here](https://github.com/TypeStrong/typedoc/blob/master/package.json#L36).

**Comment by KyleMayes** [comment]
Fixed in `v0.3.1`.

**Comment by krisztianb** [comment]
Great, thank you. 👍
